tools/reflection/ReflectionParser.cpp

```diff
@@ -329,16 +330,32 @@ ReflectionParser::ParseInstruction(const spv_parsed_instruction_t *inst) {
 
 namespace clspv {
 
-bool ParseReflection(const std::vector<uint32_t> &binary, spv_target_env env,
+bool ParseReflection(const uint32_t* spv_binary,
```

Comment:
This tool is deprecated and should not be used. APIs should parse the reflection data into their own internal data structures.

Please revert changes to this directory.

Reply:
My mistake - I did not know this tool was deprecated. Will update and push.